GG/GG/ListBox.h

```diff
@@ -323,12 +320,17 @@ class GG_API ListBox : public Control
         auto-scrolling. */
     unsigned int    AutoScrollInterval() const;
 
+    /** Return true if drops are allowed.*/
+    bool AllowingDrops();
+
+
     mutable ClearedSignalType        ClearedSignal;         /// the cleared signal object for this ListBox
     mutable BeforeInsertSignalType   BeforeInsertSignal;    ///< the before insert signal object for this ListBox
     mutable AfterInsertSignalType    AfterInsertSignal;     ///< the after insert signal object for this ListBox
     mutable SelChangedSignalType     SelChangedSignal;      ///< the selection change signal object for this ListBox
     mutable DroppedSignalType        DroppedSignal;         ///< the dropped signal object for this ListBox
     mutable DropAcceptableSignalType DropAcceptableSignal;  ///< the drop-acceptability signal object for this ListBox
+    mutable RowMovedSignalType       MovedSignal;           ///< the moved signal object for this ListBox
```

Comment:
I've changed them all.  `LeftClickedSignal` was also ambiguous, but the new `LeftClickedRowSignal` is not.